rest-service/queue/src/main/java/mb/lib/extern/model/JobStatusResponse.java

```diff
@@ -7,16 +7,16 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class JobStatusResponse
 {
-  private String status;
+  private QueueJobStatus status;
```

Comment:
Converted the internal type because I was a fool before for not storing it this way.